### ISSUE TITLE
Phrases are not always honored within advanced search

### ISSUE THREAD

**Issue by brent-hartwig** [project-lux/lux-marklogic#273]
**Problem Description:**
Phrases are not always honored within advanced search fields, specifically when an advanced search field contains a phrase and another word or phrase.  For example, searching for `"physical measurements" analysis` anywhere within works provides the same results as `physical measurements analysis`; yet, in simple search, there are fewer results for the search containing the phrase.  In the 2024-07-17 dataset, there should be 142 results when using the quotes and 1,547 results when not using quotes.

![image](https://github.com/user-attachments/assets/ebd2c4b7-31d0-4c4c-bf9f-29285088927c)

**Expected Behavior/Solution:**
Advanced search should also honor the quotes / phrases.

**Requirements:**
Fix it.

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

~- [ ] Wireframe/Mockup - Mike~
- [x] Committee discussions - Sarah
- [x] Feasibility/Team discussion - Sarah
- [x] Backend requirements - Clear
- [x] Middle tier requirements - None
- [x] Frontend requirements - None
- [x] Are new regression tests required for QA - Amy
- [x] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**
TST
Quoted https://lux-front-tst.collections.yale.edu/view/results/works?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22%5C%22physical+measurements%5C%22+analysis%22%7D%5D%7D
Is different from unquoted
https://lux-front-tst.collections.yale.edu/view/results/works?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22%5C%22physical+measurements%5C%22+analysis%22%7D%5D%7D

PRD
quoted https://lux.collections.yale.edu/view/results/works?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22%5C%22physical+measurements%5C%22+analysis%22%7D%5D%7D
Is the same as unquoted
https://lux.collections.yale.edu/view/results/works?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22physical+measurements+analysis%22%7D%5D%7D

**Dependencies/Blocks:**
This ticket neither blocks nor is blocked by another ticket.

**Related Github Issues:**
None.

**Related links:**
None

**Wireframe/Mockup:**
N/A

**Comment by brent-hartwig** [comment]
Addressed in PR https://github.com/project-lux/lux-marklogic/pull/274 which was merged into the release1.23 branch.

**Comment by brent-hartwig** [comment]
@roamye, as soon as Backend v1.23 reaches TST, the Awaiting Review label is legit.

**Comment by roamye** [comment]
@brent-hartwig - wanted to confirm that backend v1.23 has not reached tst and thus this should not be reviewed for tomorrows UAT.

**Comment by brent-hartwig** [comment]
@clarkepeterf, please review and if you like merge PR https://github.com/project-lux/lux-marklogic/pull/296 into the release1.23.3 branch.  Per @roamye's comment above, we'd need to get this to TST by tomorrow to include in UAT.  Might we be able to pull that off?  Please advise if you'd like my help.

**Comment by brent-hartwig** [comment]
@roamye, Peter deployed the corrected implementation to all non-prod environments.

**Comment by roamye** [comment]
apologies I did not realize this didn't go through the UAT process. It will need to be reviewed tomorrow as this should have gone to prioritization review first.

**Comment by clarkepeterf** [comment]
@roamye ok. This change is currently deployed in all lower envs (set to go to PRD with this release)

**Comment by roamye** [comment]
@clarkepeterf @brent-hartwig - I checked the PROD links and the quoted/unquoted do not match in number. While in TST it does. Any insight on why this is happening?

**Comment by brent-hartwig** [comment]
@roamye, I'm getting identical search result counts between TST and PROD with and without quotes.  Please verify you are selecting "Anywhere" (vs "Name")

**Comment by roamye** [comment]
@brent-hartwig - these are the searches I looked at: https://lux.collections.yale.edu/view/results/works?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22%5C%22physical+measurements%5C%22+analysis%22%7D%5D%7D (quoted) vs https://lux.collections.yale.edu/view/results/works?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22physical+measurements+analysis%22%7D%5D%7D (unquoted). For the first one it returns 142 and the second returns 1547. I have verified both links are using "Anywhere" vs "Name".

**Comment by roamye** [comment]
Looks good in prod. Closing,
